src/storage/engine/memory.rs

```diff
@@ -44,8 +44,8 @@ impl Engine for EngineBtree {
         Ok(m.get(key.raw()).cloned())
     }
 
-    fn seek(&self, _: &Context, key: &Key) -> Result<Option<KvPair>> {
-        trace!("EngineBtree: seek {}", key);
+    fn scan(&self, _: &Context, key: &Key) -> Result<Option<KvPair>> {
```

Comment:
I think `scan` need `startKey`, `endKey` and `limit`, and returns a list of KvPair.
If we need scan we better add a method `scan` along with `seek`.